docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "2.19.1"
+#------------------------------------------------------------------------------------------------------------------------------
+#- Ping Identity PingFederate
+#-
+#-     login  console
+#-     9031    9999
+#-      |       |
+#-   +---------------+
+#-   | PingFederate  |
+#-   +---------------+
+#-
+#-   +-----------------------+--------------------------------------------------------+
+#-   |  Product Console/App  |  URL                                                   |
+#-   |                       |    username: administrator                             |
+#-   |                       |    password: 2FederateM0re                             |
+#-   +-----------------------+--------------------------------------------------------+
+#-   |  PingFederate         |  https://localhost:9999/pingfederate/app               |
+#-   +-----------------------+--------------------------------------------------------+
+#-------------------------------------------------------------------------------------------------------------------------------
+
+services:
+  pingfederate:
+    image: pingidentity/pingfederate:2305
```

Comment:
I'd use a tag with the actual PingFed version here for clarity

Reply:
2305-11.2.5 added